### PR TITLE
Update readme to reflect new version

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary
Update readme to reflect new version

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,15 +49,13 @@ You can leave the Beta testing program at any time:
 
 App data is wiped from the device when a user logs out of the app. If the user is logged in when the account is deactivated, then within one minute the system logs the user out, and as a result all app data is wiped from the device.
 
-### Can I connect to multiple Mattermost servers using the mobile apps?
-
-At the moment, we only support connecting to one server at a time. If you need to connect to multiple servers, please [upvote the feature request](https://mattermost.uservoice.com/forums/306457/suggestions/10975938) so we can track demand for it. 
+### Will there be second generation apps available for tablets?
```
Comment (esethna) [inline README.md:52]: Propose remove this question. tablet support was built in v2

**Review by esethna — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (enahum) [reply README.md:52]: Agree with this comment

File: README.md [hunk 2]
```diff
@@ -51,15 +49,13 @@ You can leave the Beta testing program at any time:
...
 
-As a work around, you can install both the released "Mattermost" app and sign up to be a [tester](#testing) for the "Mattermost Beta" app so you can connect to two servers at once.
+Ipads are currently supported, but we plan to add support for tablets in the future, but the timeline depends on how many people have a need for it. If you're looking for a tablet version, please help us out by [upvoting the feature request](https://mattermost.uservoice.com/forums/306457/suggestions/20082079)!
```
Comment (enahum) [inline README.md:54]: There are no android tablets, and is iPad

**Review by enahum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Willyfrog) [reply README.md]: removed

**Review by enahum — APPROVED** [review]

**Pull request merged**